README.md

```diff
@@ -260,6 +261,43 @@ it.
 NOTE: You must ensure that the data type (e.g. Object, Array or String) for
 the replaced value is correct.  This cannot be done automatically.
 
+Hjson Files
+----------
+
+Hjson is a syntax extension to JSON. It is intended to be used like a user interface for humans, to read and edit before passing the JSON data to the machine. That means you can use it to parse JSON files but it is not intended as a replacement.
+
+You can check [Hjson's homepage](https://hjson.org) to get familiar with it and you can [try out it's syntax](https://hjson.org/try.html).
```

Comment:
english nit: it's -> its

Reply:
[its-its](http://grammarist.com/spelling/its-its/)

Reply:
(yeah, I too know the difference, but I still make that error ALL THE TIME. That's why I'm *always* watching for it. ;)

Reply:
Oh ok, did not know that, thank you. But it applies only to `it` right?

Reply:
The way I remember is the "it is or it has" rule. If you can't stick "it is" in the place of `it's`, then the correct spelling is `its`.  It's a stupid rule because English is so irregular.